projects/go-style-guide/src/app/app.component.ts

```diff
@@ -60,7 +60,8 @@ export class AppComponent {
       {
         displayName: 'GitHub',
         icon: 'code',
-        url: 'https://github.com/'
+        url: 'https://github.com/',
+        target: 'github'
```

Comment:
It allows you to specify a name for a tab basically. Scenario:

1. User clicks link to go to a new tab, browser opens a new tab and takes the user there
2. User goes back to previous tab
3. User clicks same link, instead of opening another new tab the browser just focuses on the other previously opened tab with the same target name

It's weird and we probably won't use it in Mobile, but it's behavior that exists in the Fixed world so.. 🤷

Reply:
Ah, neat. Thanks for explaining that 👍